### PR TITLE
Update websocket based ports

### PR DESCRIPTION
**Pull request by KrishnaIyer**
#### Summary

Update websocket based ports Refs https://github.com/TheThingsIndustries/lorawan-stack/pull/2402

#### Changes

- Fix protocol for LBS
- Add TH ports

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/reference/networking/_index.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ The following table lists the default ports used.
 | Management, data, events | gRPC | API key, token | 1884 | 8884 |
 | Management | HTTP | API key, token | 1885 | 8885 |
 | Backend Interfaces | HTTP | Custom | N/A | 8886 |
-| Basic Station LNS | HTTP | Auth Token, Custom | 1887 | 8887 |
+| LoRa Basic Station LNS | Web Sockets | Auth Token, Custom | 1887 | 8887 |
+| Tabs Hubs LNS | Web Sockets | Auth Token, Custom | 1888 | 8888 |
```
Comment (htdvisser) [inline doc/content/reference/networking/_index.md:24]: Shouldn't this somehow be marked as an enterprise feature?

**Review by johanstokking — APPROVED** [review]

**Comment by htdvisser** [comment]
Adding @benolayinka to see if this is the correct way to indicate enterprise-only features

**Review by htdvisser — APPROVED** [review]
LGTM, but please wait for @benolayinka (see my previous comment)

**Review by neoaggelos — APPROVED** [review]

**Comment by KrishnaIyer** [comment]
@benolayinka: Waiting for your check.

File: doc/content/reference/networking/_index.md [hunk 2]
```diff
@@ -20,7 +20,15 @@ The following table lists the default ports used.
 | Management, data, events | gRPC | API key, token | 1884 | 8884 |
 | Management | HTTP | API key, token | 1885 | 8885 |
 | Backend Interfaces | HTTP | Custom | N/A | 8886 |
-| Basic Station LNS | HTTP | Auth Token, Custom | 1887 | 8887 |
+| LoRa Basic Station LNS | Web Sockets | Auth Token, Custom | 1887 | 8887 |
```
Comment (benolayinka) [inline doc/content/reference/networking/_index.md:23]: There's a {{% lbs %}} shortcode you can use

File: doc/content/reference/networking/_index.md [hunk 3]
```diff
@@ -20,7 +20,15 @@ The following table lists the default ports used.
...
+
+### Enterprise-only Ports
```
Comment (benolayinka) [inline doc/content/reference/networking/_index.md:25]: You can use the {{< distributions-inline "Enterprise" >}} shortcode inside the table to mark this, like https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/doc/content/reference/branding/_index.md

**Review by benolayinka — APPROVED** [review]

**Pull request merged**